angular/core/models/user_model.coffee

```diff
@@ -28,6 +28,19 @@ angular.module('loomioApp').factory 'UserModel', (BaseModel, AppConfig) ->
     parentGroups: ->
       _.filter @groups(), (group) -> group.isParent()
 
+    topLevelGroups: ->
```

Comment:
It's not a new concept, it's the logic from the groups page, which is now being applied to the sidebar as well. Currently we have no way to reach the groups page, but I'm not sure if we should take it out yet.